### DISCUSSION TITLE
Possibility to edit toolbar elements

### DISCUSSION THREAD

**Discussion by kacpermigala** [dream-num/univer#1642] [Q&A | 常见问题]
Hi,

I see that there is an option to edit toolbar elements in the docs plugin, but is there the same feature available for sheets?

**Comment by jikkai** [comment]
Both sheets and docs can be modified right now, but since we haven't provided a facade API yet. We are planning to release the necessary facade API in the near future to facilitate these extensions.

**Reply by mahdi337** [reply]
Any updates?
